packages/sanity/src/desk/panes/document/DocumentOperationResults.tsx

```diff
@@ -58,15 +60,18 @@ export const DocumentOperationResults = memo(function DocumentOperationResults()
     }
 
     if (event.type === 'success' && !IGNORE_OPS.includes(event.op)) {
-      pushToast({
+      // If this is a delete operation, we want to push the toast to the root, since the current pane
+      // will be unmounted, and the toast will be lost
+      const toastFunc = event.type === 'delete' ? rootPushToast : pushToast
```

Comment:
When testing this, I do not get any toast message when deleting a document. It looks like `event` is always `undefined`. Since `DocumentOperationResults` will be unmounted when a document is deleted – will this effect ever run? Do we need to run this is a clean up function in the effect?

Reply:
I'm pretty sure it was working when I was testing, but it could be a race condition of some sort. I'll look into it.